### DISCUSSION TITLE
Icon to the right of field in create new folder dialog

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1956] [General]
I was wondering if someone could tell me what the button/icon found to the right of the text field in the create new folder dialog does?  The icon is a rectangle field with a "I" cursor.

Thank You

**Comment by Skif-off** [comment]
Allows to enable or disable auto-completion from the directory creation history.

**Comment by advice2020** [comment]
Hello again :)
Damn you're quick :)
Ahh when I first saw it, that was one of my assumptions, I went searching for the answer online/manual and forgot to come back test that button out.
Ahh that is good update, because sometimes it was helpful and other times it was not :)
Thank you for your response

**Comment by advice2020** [comment]
When I first thought that that is what this was I did try it quickly but did not notice anything.
Now that I know what it does and try it, it still does not appear to work for me.

Could be because at the moment I am using an older version of DC
Revision 11533
Commit 6a99fd2eb
does anyone know if this was fixed in newer versions?
or in general if anyone can confirm is this button works for them?
Probably just me, I plan on generating a new version soon, I will try it again but just wanted to check here just in case.
Thank You

**Comment by alexx2000** [comment]
Works fine with
```
Double Commander
Version: 1.2.0 alpha
Revision: 11685
Commit: f046c9f
Build date: 2024/11/19
Lazarus: 3.6.0.0
FPC: 3.2.2
Platform: x86_64-Linux-qt5
OS version: Ubuntu 24.04.1 LTS

Widgetset library: Qt 5.15.13, libQt5Pas 5.6.1
```

**Comment by ags1234** [comment]
A tooltip would be nice for this button. And for the "Extended syntax" checkbox.
